index.js

```diff
@@ -0,0 +1,108 @@
+const express = require("express");
+const cors = require("cors");
+const helmet = require("helmet");
+const bcrypt = require("bcryptjs");
+const knexConfig = require("./knexfile.js");
+const knex = require("knex");
+const session = require("express-session");
+const KnexSessionStore = require("connect-session-knex")(session);
+
+const db = knex(knexConfig.development);
+
+const server = express();
+
+const sessionConfig = {
+  secret: "dont-steal.my%secret!",
+  name: "someWierdAnimal",
+  httpOnly: true,
+  resave: false,
+  saveUninitialized: false,
+  cookie: {
+    secure: false,
+    maxAge: 1000 * 60 * 1
+  },
+  store: new KnexSessionStore({
+    tablename: "sessions",
+    sidfieldname: "sid",
+    knex: db,
+    createtable: true,
+    clearInterval: 1000 * 60 * 60
+  })
+};
+server.use(session(sessionConfig));
+
+server.use(express.json());
+server.use(cors(), helmet());
+
+// The Sanity Check
+server.get("/", (req, res) => {
+  res.send("Its Alive!");
+});
+
+server.get("/logout", (req, res) => {
+  if (req.session) {
+    req.session.destroy(err => {
+      if (err) {
+        res.send("You Can't Leave!!! HAHAHAHA");
+      } else {
+        res.send("See Ya");
+      }
+    });
+  }
+});
+
+server.post("/register", (req, res) => {
+  const creds = req.body;
+  const hash = bcrypt.hashSync(creds.password, 14);
```

Comment:
For development purposes, its okay, and recommended, to use a lower number of rounds, say something like 7 or 8, so you don't spend too much time waiting around. But then you'd bump it up to at least 14 for production.